src/Swarm/Game/WorldGen.hs

```diff
@@ -77,7 +99,7 @@ testWorld2 baseSeed (Coords ix@(r, c)) =
     | h `mod` 10 == 0 = (GrassT, Just (T.concat ["bit (", from (show ((r + c) `mod` 2)), ")"]))
     | otherwise = (GrassT, Nothing)
   genBiome Big Soft Artificial
-    | h `mod` 5000 == 0 = (DirtT, Just "linux")
+    | h `mod` 5000 == 0 = (DirtT, Just "Linux")
```

Comment:
Hah, I had been wondering why I have never, ever seen a `Linux`.

Reply:
Weird that it did not throw an exception. :confused: